src/config/dom.js

```diff
@@ -92,7 +92,7 @@ export default {
 
   // Does the el have child nodes
   hasContents(el) {
-    return el.hasChildNodes();
+    return !!el && el.hasChildNodes();
```

Comment:
new behavior should be tested here: https://github.com/marionettejs/backbone.marionette/blob/master/test/unit/config/dom.js#L265

Reply:
Done

Reply:
@paulfalgout can you dismiss this request?